<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Frame</class>
 <widget class="QMainWindow" name="Frame" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QTerm</string>
  </property>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;Terminal</string>
    </property>
    <addaction name="actionConsole" />
    <addaction name="separator" />
    <addaction name="actionQuick_Login" />
    <addaction name="actionAddressBook" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionFont" />
    <addaction name="separator" />
    <addaction name="actionStatusbar" />
    <addaction name="actionRuler" />
    <addaction name="actionFullscreen" />
   </widget>
   <widget class="QMenu" name="menuANSI" >
    <property name="title" >
     <string>&amp;ANSI</string>
    </property>
    <addaction name="actionNew_ANSI" />
    <addaction name="actionOpen_ANSI" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="separator" />
    <addaction name="actionPrint_Preview" />
    <addaction name="actionPrint" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="separator" />
    <addaction name="actionAuto_Copy" />
    <addaction name="actionRectangle_Selection" />
    <addaction name="actionPaste_w_Wordwrap" />
    <addaction name="separator" />
    <addaction name="actionCopy_w_Color" />
    <addaction name="actionPaste_w_Color" />
    <addaction name="separator" />
    <addaction name="actionSysbols" />
   </widget>
   <widget class="QMenu" name="menuFavatite" >
    <property name="title" >
     <string>&amp;Favatite</string>
    </property>
    <addaction name="actionAdd_to_favarites" />
    <addaction name="actionManage_Favarites" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuBBS" >
    <property name="title" >
     <string>&amp;BBS </string>
    </property>
    <addaction name="actionCopy_Article" />
    <addaction name="actionAnti_idle" />
    <addaction name="actionAuto_Reply" />
    <addaction name="actionSupport_Mouse" />
    <addaction name="actionBeep_on_message" />
    <addaction name="actionView_Message" />
    <addaction name="actionArticle_Manager" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>&amp;Options</string>
    </property>
    <addaction name="actionClipboard_Encoding" />
    <addaction name="separator" />
    <addaction name="actionDefault_Session_Setting" />
    <addaction name="actionPreference" />
    <addaction name="separator" />
    <addaction name="actionConfigure_Shortcuts" />
    <addaction name="actionConfigure_Toolbars" />
   </widget>
   <widget class="QMenu" name="menuScript" >
    <property name="title" >
     <string>&amp;Script</string>
    </property>
    <addaction name="actionRun" />
    <addaction name="actionStop" />
    <addaction name="actionDebug_Console" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionContents" />
    <addaction name="actionWhat_s_this" />
    <addaction name="separator" />
    <addaction name="actionAbout_QTerm" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuWindow" >
    <property name="title" >
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuANSI" />
   <addaction name="menuBBS" />
   <addaction name="menuEdit" />
   <addaction name="menuView" />
   <addaction name="menuFavatite" />
   <addaction name="menuOptions" />
   <addaction name="menuScript" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="TerminalBar" >
   <property name="windowTitle" >
    <string>Terminal</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>1</number>
   </attribute>
  </widget>
  <widget class="QToolBar" name="EditBar" >
   <property name="windowTitle" >
    <string>Edit</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>2</number>
   </attribute>
  </widget>
  <widget class="QToolBar" name="ANSIBar" >
   <property name="windowTitle" >
    <string>ANSI Edit</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <widget class="QToolBar" name="PagerBar" >
   <property name="windowTitle" >
    <string>Pager</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <action name="actionQuick_Login" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/quicklogin.png</iconset>
   </property>
   <property name="text" >
    <string>Quick Login</string>
   </property>
   <property name="whatsThis" >
    <string>This provides an express way to connect.</string>
   </property>
  </action>
  <action name="actionAddressBook" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/addressbook.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;AddressBook</string>
   </property>
  </action>
  <action name="actionNew_ANSI" >
   <property name="text" >
    <string>New ANSI</string>
   </property>
  </action>
  <action name="actionOpen_ANSI" >
   <property name="text" >
    <string>Open ANSI</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="iconText" >
    <string>Quit</string>
   </property>
   <property name="toolTip" >
    <string>Quit</string>
   </property>
   <property name="whatsThis" >
    <string>This will close all windows. You will be prompted to confirm disconnent active sessions or save modified files.</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionConsole" >
   <property name="text" >
    <string>New &amp;Console</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/copy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/paste.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionCopy_w_Color" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/copywcolor.png</iconset>
   </property>
   <property name="text" >
    <string>Copy w Color</string>
   </property>
  </action>
  <action name="actionPaste_w_Color" >
   <property name="text" >
    <string>Paste w Color</string>
   </property>
  </action>
  <action name="actionPaste_w_Wordwrap" >
   <property name="text" >
    <string>Paste w Wordwrap</string>
   </property>
  </action>
  <action name="actionRectangle_Selection" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/rectselect.png</iconset>
   </property>
   <property name="text" >
    <string>Rectangle Selection</string>
   </property>
  </action>
  <action name="actionAuto_Copy" >
   <property name="text" >
    <string>&amp;Auto Copy</string>
   </property>
  </action>
  <action name="actionClipboard_Encoding" >
   <property name="text" >
    <string>Clipboard Encoding</string>
   </property>
  </action>
  <action name="actionSysbols" >
   <property name="text" >
    <string>Sysbols</string>
   </property>
  </action>
  <action name="actionFullscreen" >
   <property name="text" >
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionDefault_Session_Setting" >
   <property name="text" >
    <string>Default Session Setting</string>
   </property>
  </action>
  <action name="actionPreference" >
   <property name="text" >
    <string>&amp;Preference</string>
   </property>
  </action>
  <action name="actionAdd_to_favarites" >
   <property name="text" >
    <string>Add to favarites</string>
   </property>
  </action>
  <action name="actionManage_Favarites" >
   <property name="text" >
    <string>Manage Favarites ...</string>
   </property>
  </action>
  <action name="actionCopy_Article" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/copyarticle.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy Article</string>
   </property>
  </action>
  <action name="actionAuto_Reply" >
   <property name="text" >
    <string>&amp;Auto Reply</string>
   </property>
  </action>
  <action name="actionSupport_Mouse" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/mouse.png</iconset>
   </property>
   <property name="text" >
    <string>Support &amp;Mouse</string>
   </property>
  </action>
  <action name="actionBeep_on_message" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/beep.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Beep on message</string>
   </property>
  </action>
  <action name="actionConfigure_Shortcuts" >
   <property name="text" >
    <string>Configure Shortcuts</string>
   </property>
  </action>
  <action name="actionView_Message" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/message.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;View Message </string>
   </property>
  </action>
  <action name="actionRun" >
   <property name="text" >
    <string>Run...</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionContents" >
   <property name="text" >
    <string>Contents...</string>
   </property>
  </action>
  <action name="actionAbout_QTerm" >
   <property name="text" >
    <string>About QTerm</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="text" >
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_Preview" >
   <property name="text" >
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionAnti_idle" >
   <property name="text" >
    <string>Anti-&amp;idle</string>
   </property>
  </action>
  <action name="actionDebug_Console" >
   <property name="text" >
    <string>Debug Console</string>
   </property>
  </action>
  <action name="actionWhat_s_this" >
   <property name="text" >
    <string>What's this?</string>
   </property>
  </action>
  <action name="actionQTerm_Online" >
   <property name="text" >
    <string>QTerm Online</string>
   </property>
  </action>
  <action name="actionTheme" >
   <property name="text" >
    <string>&amp;Theme</string>
   </property>
  </action>
  <action name="action_Font" >
   <property name="icon" >
    <iconset resource="../frame.qrc" >:/pic/fonts.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Font</string>
   </property>
  </action>
  <action name="actionConfigure_Toolbars" >
   <property name="text" >
    <string>Configure Toolbars</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionRuler" >
   <property name="text" >
    <string>Ruler</string>
   </property>
  </action>
  <action name="actionFont" >
   <property name="text" >
    <string>Font</string>
   </property>
  </action>
  <action name="actionStatusbar" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Statusbar</string>
   </property>
   <property name="toolTip" >
    <string>Show/Hide Statusbar</string>
   </property>
   <property name="whatsThis" >
    <string>Show/Hide Statusbar</string>
   </property>
  </action>
  <action name="actionArticle_Manager" >
   <property name="text" >
    <string>Article Manager</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../frame.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionAbout_Qt</sender>
   <signal>activated()</signal>
   <receiver>_layout</receiver>
   <slot></slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
